frontend/service.bal

```diff
@@ -207,7 +212,7 @@ service / on new http:Listener(9098) {
         response.setJsonPayload({
             user_currency: [request.currency, currencyLogo(request.currency)],
             currencies: supportedCurrencies,
-            cart_size: cart.items.length(),
+            cart_size: check getCartSize(cart),
```

Comment:
Is there any reason for this change?

Reply:
2 items from one products counted as 1 in our previous impl